djangocms_versioning/models.py

```diff
@@ -99,12 +99,19 @@ def delete(self, using=None, keep_parents=False):
         """Deleting a version deletes the grouper
         as well if we are deleting the last version."""
 
+        def get_grouper_name(ContentModel, grouper):
```

Comment:
Wondering if this should be a helper in helpers.py, get_grouper_name_from_content_model(). I guess this depends on whether it could be used again in this package or by others.

Reply:
I haven't seen the need for this elsewhere so it seems like this would be
premature optimization. I made it its own method because it makes things
more readable.